scripts/percona-release.sh

```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
   exit 1
 fi
 #
+
+PRODUCTS_ABBREVIATIONS=("pdmdb" "pdps" "pdpxc" "pmm" "ppg11" "ppg12" "ppg13" "ppg14" "psmdb" "px")
+
+function sort_array {
+  ARRAY=$1
+  local flagforabbr="pdmdb"
+  PRODUCTS=()
+  OTHER_CURRENT_ALIASES=()
+  for element in ${ARRAY[@]}
+    do
+      counter=1
+      for abbr in ${PRODUCTS_ABBREVIATIONS[@]}
+        do
+	  if ! [[ "${element//-/}" =~ ^$abbr.*$ ]]; then
+            if [[ $counter == ${#PRODUCTS_ABBREVIATIONS[@]} ]]; then
+              OTHER_CURRENT_ALIASES+="$element "
+            else
+              counter=$((counter+1))
+            fi
+          else
+            if ! [[ $flagforabbr == $abbr ]]; then
+              flagforabbr=$abbr
+              echo ${PRODUCTS}
+              PRODUCTS=()
+              PRODUCTS+="$element "
+              break
+	    else
+              PRODUCTS+="$element "
+	    fi
+          fi
+        done
+    done
+  echo -e ${PRODUCTS}
+  echo -e ${OTHER_CURRENT_ALIASES}
+}
+
+function get_repos_from_site {
+  CURRENT_REPOS=$(curl -s https://repo.percona.com | tail -n +28  | grep href | grep -v https | awk -Fhref=\" '{print $2}' | awk -F\/ '{print $1}') 
```

Comment:
what should be in case repo is unavailable?

Reply:
I have restored variable REPOSITORIES with default list of repos in such case.